grayskull/strategy/pypi.py

```diff
@@ -288,7 +288,7 @@ def get_pypi_metadata(config: Configuration) -> dict:
         "url": info.get("home_page"),
         "license": info.get("license"),
         "source": {
-            "url": config.url_pypi + "/packages/source/{{ name[0] }}/{{ name }}/"
+            "url": config.url_pypi + f"/packages/source/{config.name[0]}/{config.name}/"
```

Comment:
These will always be normalized (even for historic builds), with the most-widely used implementation being [`packaging.utils.canonicalize_name`](https://packaging.pypa.io/en/stable/utils.html#packaging.utils.canonicalize_name).

Reply:
that is a good idea, grayskull still tries to canonicalize the names, but having that implementation to use out of the box might be better indeed

Reply:
Should this be stored as a key for reuse?